pkg/config/loader.go

```diff
@@ -49,12 +50,12 @@ func LoadConfig() (*Config, error) {
 
 	// 1 - Read in the config file.
 	if err = viper.ReadInConfig(); err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
```

Comment:
Actually showing the stack trace only happens if you run sonobuoy with `--debug`, otherwise you still just get the error message, annotated with the source line of wherever you chose to log it (same behavior as now.)

So to make `--debug` work in general, I'm using WithStack so that the error has the source file:line "tagged" onto it at the place where it happened.  In this case, since we called WithStack on loader.go:53, the stack trace will show loader.go:53 in it.  If we instead passed the error up untouched until it was logged in (say) main.go, the stack trace would just show main.go in it, which is less helpful.